app/javascript/components/GuitarInfo.js

```diff
@@ -7,10 +8,59 @@ import {
   Card,
   CardTitle,
   CardBody,
-  CardFooter
+  CardFooter,
+  Button,
+  Modal,
+  ModalVariant,
+  Form,
+  FormGroup,
+  TextInput
 } from '@patternfly/react-core';
+import { getGuitarUrlWithId } from './constants.js';
+
+const GuitarInfo = ({ id, name, url, price, description }) => {
+  const [isFormModalOpen, toggleFormModal] = useReducer(
+    (isFormModalOpen) => !isFormModalOpen,
+    false
+  );
+
+  // Form Attributes useStates():
+  const [nameValue, setNameValue] = useState(name);
+  const [urlValue, setUrlValue] = useState(url);
+  const [priceValue, setPriceValue] = useState(price);
+  const [descriptionValue, setDescriptionValue] = useState(description);
+
+  const editGuitar = () => {
+    // Q: Do I need to set the from values here as well as initialize it above?
+
+    // Populate fields of form with the edited guitar's attributes:
```

Comment:
@MariaAga Following our conversation, the following setters are required in this case:
The user clicks on the edit button, changes some form attributes, and then decides to NOT submit the form
In this case, the guitar is NOT updated in the DB with the new values, but the corresponding useState does change and holds the new values the user inserted
In case the user chooses to click on the edit button again, the form presents the current values of the useStates.
Therefore, the form would present the wrong values (It is supposed to present the values of the state of the guitar in the gallery, as it is currently in the DB, but instead it presents the last edited values that were in the form)

In order to keep it simple, I think it would be best to keep it like it is now
WYT? @MariaAga